docs/tools/vdb_table/data/pinecone.json

```diff
@@ -54,12 +54,12 @@
   },
   "sparse_vectors": {
     "support": "full",
-    "source_url": "",
-    "comment": ""
+    "source_url": "https://docs.pinecone.io/docs/hybrid-search",
+    "comment": "Supports all sparse vectors, including BM25 and native support for SPLADE vectors"
   },
   "bm25": {
-    "support": "none",
-    "source_url": "",
+    "support": "full",
```

Comment:
This column is to indicate that the DB has native support to compute BM25 scores. I don't think that is present in Pinecone

Reply:
I understood to mean support for searching with bm25 vecs, modified